workspaces/quay/plugins/quay/src/components/QuayRepository/QuayRepository.tsx

```diff
@@ -16,16 +16,21 @@
 import { Link, Progress, Table } from '@backstage/core-components';
 import { configApiRef, useApi } from '@backstage/core-plugin-api';
 
+import { Box, Typography } from '@material-ui/core';
+
 import { useRepository, useTags } from '../../hooks';
 import { useQuayViewPermission } from '../../hooks/useQuayViewPermission';
 import PermissionAlert from '../PermissionAlert/PermissionAlert';
-import { columns, useStyles } from './tableHeading';
+import { columns } from './tableHeading';
 
 type QuayRepositoryProps = Record<never, any>;
 
 export function QuayRepository(_props: QuayRepositoryProps) {
   const { repository, organization } = useRepository();
-  const classes = useStyles();
+  const authTokenDocs =
```

Comment:
I would store that as a const as docs tend to evolve overtime

Reply:
Good idea, like this? -> https://github.com/backstage/community-plugins/pull/4370/commits/0421f0044d270479b453d4229531718c6f750f8d